analysis/index.js

```diff
@@ -59,20 +59,26 @@ function analysis (traceEventReader, processStatReader) {
       }
     }
 
-    const issues = {
-      'delay': analyseDelay(processStatSubset, traceEventSubset),
-      'cpu': analyseCPU(processStatSubset, traceEventSubset),
-      'memory': analyseMemory(processStatSubset, traceEventSubset),
-      'handles': analyseHandles(processStatSubset, traceEventSubset)
-    }
-    const category = issueCategory(issues, {detectNoise: true})
+    const p = Promise.all([
```

Comment:
I would `await`  here. Can you also refactor this function? Using both `async.parallel` and `Promise.all` seems a bit too hard to read.

Also, the function is too long.